test/test_table.cpp

```diff
@@ -3488,7 +3488,8 @@ NONCONCURRENT_TEST(Table_object_timestamp)
     }
 }
 
-NONCONCURRENT_TEST(Table_object_search)
+// NONCONCURRENT_TEST(Table_object_search)
+TEST(Table_object_search)
```

Comment:
Why not `NONCONCURRENT_TEST`?

Reply:
Also, can we restore the other tests where we have those old arrays now?

Reply:
Sorry, that was a bug